src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2097,6 +2097,66 @@ End Class</Document>)
             End Using
         End Function
 
+        <WorkItem(49632, "https://github.com/dotnet/roslyn/pull/49632")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function CompletionEnumTypeAndValuesInDifferentNamespaces() As Task
+            Using state = TestStateFactory.CreateVisualBasicTestState(
+                              <Document>
+Namespace A
+    Public Enum Colors
+        Red
+        Green
+    End Enum
+End Namespace
+
+Namespace B
+    Class Program
+        Private Shared Sub M(color as A.Colors)
+        End Sub
+
+        Private Shared Sub Main()
+             M($$)
+        End Sub
+    End Class
+End Namespace
+</Document>)
+                state.SendInvokeCompletionList()
+                Await state.AssertCompletionItemsDoNotContainAny("A.Colors", "Colors")
```

Comment:
This is different from C#. In C# the Enum type is listed with namespace "A.Colors" (see test above). I think this is also different from what the behavior is today, without this PR. I think today in VB this is listed as "Colors". But I'm not sure what the source of this "Colors" is today, because in VB lots of types from other namespaces are listed, so it might be "IntelliCode" that lists "Colors" today (and will so in the future).

Reply:
IntelliCode does not run in our tests.

Note: we cannot change behavior here.  It is important for muscle memorythat we still preselect the same item that we used to.  thanks!

Reply:
I look more closely and I think the behavior today is broken and the behavior of this PR is "okayish" (example uses the unimported `Microsoft.VisualBasic.FileIO` namespace).

### Today

![PNtnkVbJHm](https://user-images.githubusercontent.com/24472128/102513783-f3368380-408b-11eb-8938-8bca6ba56630.gif)

`FieldType` is from an unimported namespace and listed as `FieldType`. `<tab>`ing it, inserts *FieldType*, which is wrong and should be `FileIO.FieldType`

### Now (this PR)

![TQGVTdm6Vx](https://user-images.githubusercontent.com/24472128/102514337-a2735a80-408c-11eb-991d-cb2aadde9417.gif)

`FieldType` is no longer offered, as the test above also shows.

So this PR removes a broken suggestion. Note: This is only true for unimported namespaces. If the enum type is in scope, it is suggested as expected.